### PR TITLE
Feature/compile on switch

### PR DESCRIPTION
**Pull request by BrahimS**
Migrate all the logics of the text editor to PageProductChild.vue and compile on switch.
See the Demo :  https://nextcloud.ditrit.io/index.php/s/NfF9EFJ45FENqM8

TODO in the futur: 
1 - Update to composition API
2 - Create a composable file with all the logics for text editor.

### CODE REVIEW

File: frontend/src/pages/Products/PageProductChild.vue [hunk 1]
```diff
@@ -90,20 +104,166 @@ export default defineComponent({
...
+			valueEditor: null,
+		};
+	},
+	created() {
+		this.worker = new Worker(
+			new URL("src/components/Monaco/iactor.js", import.meta.url)
+		);
+		this.worker.onmessage = function (event) {
+			this.iactorDatas = event.data;
+			console.log("this.iactorDatas: ", this.iactorDatas);
```
Comment (Zorin95670) [inline frontend/src/pages/Products/PageProductChild.vue]: Remove all `console.log`.

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Products/PageProductChild.vue (hunk 1 above)
Comment (BrahimS) [reply frontend/src/pages/Products/PageProductChild.vue]: ok

**Review by Mari0nV — CHANGES_REQUESTED** [review]
There is a bug when we compile a file twice:

https://user-images.githubusercontent.com/23618241/177179493-f5485651-1fab-4215-bdc6-0df942ce2c65.mp4

**Review by AlexisNeveu — APPROVED** [review]

**Comment by BrahimS** [comment]
@marion. voila le bug est fixé : https://nextcloud.ditrit.io/index.php/s/bqcX7R2aB9jg9qL

**Comment by Mari0nV** [comment]
I see other bugs: 
- When changing the content of Text, the Model 3D does not show the changes immediately.
- When adding a component in the Model 3D, it does not change the Text.

I tested all of this in demo-4-july on Mozilla and these problems were not there.

https://user-images.githubusercontent.com/23618241/177187276-6e5a9a14-11e5-4ac9-bfa8-d65dbc42152e.mp4

**Comment by Mari0nV** [comment]
I still see a bug that I don't have on demo-4-july:

https://user-images.githubusercontent.com/23618241/177828122-311bb48e-b4ab-4c94-a42c-b2b9202fe479.mp4

The aws instance is not displayed in the Model 3D view although it's present on the Text view

**Comment by BrahimS** [comment]
[Demo](https://user-images.githubusercontent.com/7687826/177835713-8f9c9140-fdcb-435b-a68f-a5e706ca9e11.webm)

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**